mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -261,18 +303,18 @@ func initializeReleaseDiffTest(c resource.TestCase, testName string) resource.Te
 			replacementSteps = append(replacementSteps, testStep)
 		}
 	}
-
-	c.Steps = replacementSteps
-
-	return c
+	return replacementSteps
```

Comment:
I'd recommend just keeping `c.Steps = replacementSteps` with no return value, since the steps are being mutated anyway